### PR TITLE
Maintainability upgrades to user profile

### PR DESCRIPTION
**Pull request by melvinhernandez**
Pivotal Tracker Chore: [#165627145](https://www.pivotaltracker.com/story/show/165627145)

# Problem
Currently there is a 25 mins maintainability tech-debt issue on Code Climate.

# Solution
Change the if/else conditions to ternary operators.

### CODE REVIEW

File: app/controllers/profile_controller.rb [hunk 1]
```diff
@@ -22,21 +22,9 @@ def edit
...
-    else
-      @profile.update_attribute(:graduate, false)
-    end
-    if params[:international]
-      @profile.update_attribute(:international, true)
-    else
-      @profile.update_attribute(:international, false)
-    end
+    @profile.update_attribute(:transfer, params[:transfer] ? true : false)
+    @profile.update_attribute(:transfer, params[:graduate] ? true : false)
```
Comment (tonyli537) [inline app/controllers/profile_controller.rb]: Need to change the attribute that is being updated :transfer -> :graduate, also the next line

**Review by tonyli537 — CHANGES_REQUESTED** [review]

File: app/controllers/profile_controller.rb (hunk 1 above)
Comment (melvinhernandez) [reply app/controllers/profile_controller.rb]: Good catch!! Thanks!

**Review by tonyli537 — APPROVED** [review]
good to go

**Comment by Hekaiyo** [comment]
Short change, not much to say. Looks good.

**Review by Hekaiyo — APPROVED** [review]
Oops, I left a comment instead of a review. Still looks good.

**Pull request merged**